bottles/frontend/windows/main_window.py

```diff
@@ -174,10 +174,22 @@ def set_manager(result, error=None):
                 title=_("Library")
             ).set_icon_name("emote-love-symbolic")
 
+            self.settings.bind(
+                "startup-view",
+                self.stack_main,
+                "visible-child-name",
+                Gio.SettingsBindFlags.DEFAULT
+            )
+
             self.page_list.search_bar.set_key_capture_widget(self)
             self.btn_search.bind_property('active', self.page_list.search_bar, 'search-mode-enabled',
                                           GObject.BindingFlags.BIDIRECTIONAL)
-            self.stack_main.set_visible_child_name("page_list")
+
+            if child := self.stack_main.get_child_by_name(self.settings.get_string("startup-view")):
+                self.stack_main.set_visible_child(child)
+            else:
+                self.stack_main.set_visible_child_name("page_list")
+
```

Comment:
This is actually obsolete.

Reply:
I would still leave it in, though. This prevents the application from "hanging" when a user manually enters an invalid gsetting value for whatever reason. I will create the bin binding after that though.

Reply:
I agree